### ISSUE TITLE
Disabling certain jsx-a11y rules.

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#307]
**Scenario**
Currently, if we add Aria roles to elements that have that same semantic meaning, e.g. a role list on a unordered list element:
```
<ul role="list">
    <li role="listitem">Item 1</li>
</ul>
```
the `jsx-a11y` eslint` plugin gives these errors:

```
  22:5  error  The element ul has an implicit role of list. Defining this explicitly is redundant and should be avoided      jsx-a11y/no-redundant-roles
  23:7  error  The element li has an implicit role of listitem. Defining this explicitly is redundant and should be avoided  jsx-a11y/no-redundant-roles
```
We support Assistive Tech, some of which is older and does not understand some semantic HTML elements.  This means we will need to also add the Aria roles. 

**Task**
We should check to see what is covered by `jsx-a11y/no-redundant-roles` and discuss on this issue we'd be OK with disabling this rule. 

cc @greenc05

**Comment by greenc05** [comment]
If this rule is removed, we should then go back and add role list and listitem to the footer component

**Comment by sareh** [comment]
This does not seem to be an issue anymore. This is because we are using StyleComponents and the linting does not recognise that `<StyledListItem role="listitem">` is the same as `<li  role="listitem">` but with extra styles. Therefore, the linting does not error on the `StyledListItem` line. 

Closing as we don't need to fix this any longer.

**Comment by greenc05** [comment]
Interesting @sareh I wonder if the rest of the linting rules work...
